otcdocstheme/theme/otcdocs/static/css/otcdocstheme.css

```diff
@@ -879,3 +879,8 @@ pre .cl {
 .btn-info {
   background-color: var(--dt-color-teal-30);
 }
+
+/* Open Dropdown on Hover */
+.dropdown:hover .dropdown-menu {
```

Comment:
Here I am not really sure - i.e. K8 is not doing that and I often really struggle with auto-opening things on mobile - this is killing UX and not improving it

Reply:
I looked at the OpenStack Docu
https://docs.openstack.org/xena/

However I think that is a personal design decision. I would like a vote for that and depending on that we let it on click-only or make it on hover